src/Uno.UI/UI/Xaml/FrameworkElement.wasm.cs

```diff
@@ -523,24 +523,22 @@ private static void OnGenericPropertyUpdated(object dependencyObject, Dependency
 		/// <summary>
 		/// If corresponding feature flag is enabled, set layout properties as DOM attributes to aid in debugging.
 		/// </summary>
-		private void UpdateDOMProperties()
+		private protected override void UpdateDOMProperties()
 		{
 			if (FeatureConfiguration.UIElement.AssignDOMXamlProperties && IsLoaded)
 			{
-				SetXamlProperty(nameof(Margin), Margin);
```

Comment:
You should prefix the property to make sur the user understands it's XAML related.

Reply:
All properties are prefixed with 'xaml'